docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -146,6 +156,26 @@ pulp_python, and so forth. Any plugins in folders that start with ``pulp_`` that
 in your host machine's code folder alongside the Pulp platform repository should have been installed
 and configured for virtualenv.
 
+Using Vagrant
```

Comment:
This can be rephrased to

"The Vagrant environment provides some useful built-in commands by default"